maintainer/benchmarks/lb.py

```diff
@@ -105,6 +108,8 @@
 print(f"LB shape: {lb_grid}")
 print(f"LB agrid: {agrid:.3f}")
 
+blocks_per_mpi_rank = args.blocks_per_mpi_rank
+
```

Comment:
This alias is not really needed, you can use `args.blocks_per_mpi_rank` directly at line 153.

Reply:
I have rewritten it as per your comment.